components/pricing_modal/content.tsx

```diff
@@ -0,0 +1,346 @@
+            'open_learn_more_trial_modal',
+        );
+        props.onHide();
+        dispatch(openModal({
+            modalId: ModalIdentifiers.LEARN_MORE_TRIAL_MODAL,
+            dialogType: LearnMoreTrialModal,
+        }));
+    };
+
+    return (
+        <div className='Content'>
+            <div className='self-hosted-alert'>
+                <h1>{formatMessage({id: 'pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</h1>
+                <a
+                    onClick={() =>
+                        trackEvent(
+                            TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                            'click_looking_to_self_host',
+                        )
+                    }
+                    href={CloudLinks.DEPLOYMENT_OPTIONS}
+                    rel='noopener noreferrer'
+                    target='_blank'
+                >{formatMessage({id: 'pricing_modal.reviewDeploymentOptions', defaultMessage: 'Review deployment options'})}</a>
+            </div>
+            <Modal.Header className='PreTrialPricingModal__header'>
+                <div className='header_lhs'>
+                    <h1 className='title'>
+                        {formatMessage({id: 'pricing_modal.title', defaultMessage: 'Select a plan'})}
+                    </h1>
+                    <div>{'Choose a plan to get started'}</div>
+                </div>
+                <button
+                    id='closeIcon'
+                    className='icon icon-close'
+                    aria-label='Close'
+                    title='Close'
+                    onClick={props.onHide}
+                />
+            </Modal.Header>
+            <Modal.Body className='PreTrialPricingModal__body'>
+                <div className='self-hosted-svg-left'>
+                    <LadySvg/>
+                </div>
+                <div className='self-hosted-svg-right'>
+                    <ManSvg/>
+                </div>
+                <Card
+                    topColor='#9DA7B8'
+                    plan='Starter'
+                    price={formatMessage({id: 'pricing_modal.price.free', defaultMessage: 'Free'})}
+                    briefing={{
+                        title: formatMessage({id: 'pricing_modal.briefing.starter', defaultMessage: 'Starter comes with...'}),
+                        items: [
+                            formatMessage({id: 'pricing_modal.briefing.storage', defaultMessage: '{storage}GB file storage'}, {storage: '10'}),
+                            formatMessage({id: 'pricing_modal.briefing.starter.messageHistory', defaultMessage: 'Limited message history'}),
+                            formatMessage({id: 'pricing_modal.briefing.starter.boardCards', defaultMessage: '{boards} Boards cards per server'}, {boards: '500'}),
+                            formatMessage({id: 'pricing_modal.briefing.starter.teamCount', defaultMessage: 'Limited to one team'})],
```

Comment:
Do we also need to list the apps limit (calls?) for Starter here? @johndavidlugtu 
<img width="1289" alt="Screen Shot 2022-05-17 at 8 03 22 PM" src="https://user-images.githubusercontent.com/936315/168931382-fc9413ef-bab9-438b-9b7b-5b5fc45a123d.png">

Reply:
Also @itao ^^

Reply:
Yes we will. Checking with Product Marketing to see if they already have a feature list they'll use on mattermost.com/pricing. We'll only list the key features in this pricing modal that makes upgrading really attractive.

Will report back

Reply:
@justinegeffen ^^ copy changes?

Reply:
Thanks, @cpoile! 

AFAIK the Calls limits will be added to the Pricing page as it's linked to plans. 

For this modal we could look at adding 

"1:1 DM calls" for Starter
and 
"1:1 DM, GM, and Channel calls limited to 8 people." for Professional and Enterprise.

Reply:
@justinegeffen here's the updated copy for the pricing modal by the way: https://docs.google.com/document/d/1i9iuyyV-PyuhIw19gxnCEbFfe6IE2fnWOoMThiz926o/edit#heading=h.x9s6x3i7xttx

I think we covered all our bases, but feel free to let me know if you think otherwise